src/ctapipe/monitoring/__init__.py

```diff
@@ -2,7 +2,12 @@
 Module for handling monitoring data.
 """
 from .aggregator import PlainAggregator, SigmaClippingAggregator, StatisticsAggregator
-from .interpolation import Interpolator, PointingInterpolator
+from .interpolation import (
+    ChunkInterpolator,
```

Comment:
Add final child classes here

Reply:
@ctoennis please add the child classes, your docs are failing because of this omission I believe:
```
Handler <function process_automodsumm_generation at 0x7f46067e34c0> for event 'builder-inited' threw an exception (exception: module 'ctapipe.monitoring' has no attribute 'FlatFieldInterpolator')
```